src/Compute/Compute/help/Set-AzVMSqlServerExtension.md

```diff
@@ -28,42 +28,42 @@ The **Set-AzVMSqlServerExtension** cmdlet sets the AzureSQL Server extension on
 ### Example 1: Set automatic patching settings on a virtual machine
 ```powershell
 $AutoPatchingConfig = New-AzVMSqlServerAutoPatchingConfig -Enable -DayOfWeek "Thursday" -MaintenanceWindowStartingHour 11 -MaintenanceWindowDuration 120 -PatchCategory "Important"
-Get-AzVM -ServiceName "Service02" -Name "VirtualMachine11" | Set-AzVMSqlServerExtension -AutoPatchingSettings $AutoPatchingConfig | Update-AzVM
+Get-AzVM -Name "VirtualMachine11" | Set-AzVMSqlServerExtension -AutoPatchingSettings $AutoPatchingConfig | Update-AzVM
 ```
 
 The first command creates a configuration object by using the **New-AzVMSqlServerAutoPatchingConfig** cmdlet.
 The command stores the configuration in the $AutoPatchingConfig variable.
-The second command gets the virtual machine named VirtualMachine11 on the service named Service02 by using the Get-AzVM cmdlet.
+The second command gets the virtual machine named VirtualMachine11 by using the Get-AzVM cmdlet.
 The command passes that object to the current cmdlet by using the pipeline operator.
 The current cmdlet sets the automatic patching settings in $AutoPatchingConfig for the virtual machine.
 The command passes the virtual machine to the Update-AzVM cmdlet.
 
 ### Example 2: Set automatic backup settings on a virtual machine
 ```powershell
-$AutoBackupConfig = New-AzVMSqlServerAutoBackupConfig -Enable -RetentionPeriod 10 -StorageUri $StorageUrl -StorageKey $StorageAccountKeySecure
-Get-AzVM -ServiceName "Service02" -Name "VirtualMachine11" | Set-AzVMSqlServerExtension -AutoBackupSettings $AutoBackupConfig | Update-AzVM
+$AutoBackupConfig = New-AzVMSqlServerAutoBackupConfig -Enable -RetentionPeriodInDays 10 -StorageUri $StorageUrl -StorageKey $StorageAccountKeySecure
+Get-AzVM -Name "VirtualMachine11" | Set-AzVMSqlServerExtension -AutoBackupSettings $AutoBackupConfig | Update-AzVM
 ```
 
 The first command creates a configuration object by using the **New-AzVMSqlServerAutoBackupConfig** cmdlet.
 The command stores the configuration in the $AutoBackupConfig variable.
-The second command gets the virtual machine named VirtualMachine11 on the service named Service02, and then passes it to the current cmdlet.
+The second command gets the virtual machine named VirtualMachine11, and then passes it to the current cmdlet.
 The current cmdlet sets the automatic backup settings in $AutoBackupConfig for the virtual machine.
 The command passes the virtual machine to the Update-AzVM cmdlet.
 
 ### Example 3: Disable a SQL Server extension on a virtual machine
 ```powershell
-Get-AzVM -ServiceName "Service03" -Name "VirtualMachine08" | Set-AzVMSqlServerExtension -Disable
+Get-AzVM -Name "VirtualMachine08" | Set-AzVMSqlServerExtension Disable
```

Comment:
1. I think it can be replaced by -ResourceGroupName  
2. What does "Disable" assigned to?